### PR TITLE
Release notes for 0.16.2 and try to increment sleep for PyPI delay

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
       - run: python -m pip install --upgrade pip
       - run: python -m pip install -r requirements/ci.txt
       - run: |
-          sleep 30  # PyPI upload have a small delay until visible, avoid workflow fails in release builds
+          sleep 60  # PyPI upload have a small delay until visible, avoid workflow fails in release builds
```
Comment (jl-wynen) [inline .github/workflows/docs.yml:41]: Can this be run in a loop, trying every 30s or so to download the package? Could use `pip download scipp`. Once that succeeds, we can call tox which can then install the package and doesn't even have to de-download it (hopefully).

Comment (SimonHeybrock) [reply .github/workflows/docs.yml:41]: Let us see if this work, if it does not I will try your suggestion.

File: docs/about/release-notes.rst [hunk 2]
```diff
@@ -35,6 +35,14 @@ Release Notes
    and Jan-Lukas Wynen :sup:`a`
 
 
+v0.16.2 (August 2022)
+---------------------
+
+Bugfixes
+~~~~~~~~
+
+* Fix for update to ``pythreejs-2.4.0``.
```
Comment (SimonHeybrock) [inline docs/about/release-notes.rst:44]:
```suggestion
* Fix for ``pythreejs-2.4.0``.
* Fix ipython display for ``ipywidgets>=8``
```

**Review by jl-wynen — APPROVED** [review]

File: .github/workflows/docs.yml (hunk 1 above)
Comment (SimonHeybrock) [reply .github/workflows/docs.yml:41]: @jl-wynen You were right, of course. Could you try to implement something along the lines of your suggestion? Thanks!

**Pull request merged**